index.html

```diff
@@ -275,5 +275,54 @@ <h1>About</h1>
                 </div>
             </div>
         </section>
+
+        <!-- Contact form -->
+        <section>
+            <!-- Photo at the top -->
+            <div class="contain-img1">
+                <img class="photo-2 " src="/assets/image_geometry_1.png" alt="Background photo">
+            </div>
+
+            <!-- Get stared -->
+            <div class="submit-form">
+                <h1 class="main-name">Get started</h1>
+                <p class="main-text">Lorem ipsum dolor sit, 
+                    amet consectetur adipisicing elit. Itaque, alias!
+                </p>
+
+                <!-- Inputs area -->
+                <form class="inputs-group" 
+                action="https://formspree.io/f/mrgjwpka"
+                method="POST"
+                enctype="multipart/form-data" id="contact">
+                    <input type="text"  name="name" placeholder="Your name" class="name-input" maxlength="30" required>
+                    <input type="email" name="email" placeholder="Your email" class="email-input" required>
+                    <input type="text"  name="Additional-text" placeholder="Enter your message here..." class="message-input" maxlength="500" required>
```

Comment:
- [OPTIONAL] I strongly suggest using `<textarea>` tag for message input instead of using `<input>` tag.

> The `<textarea>` tag defines a multi-line text input control.
> The `<textarea>` element is often used in a form, to collect user inputs like comments or reviews. 👌